bazel-java-deps.bzl

```diff
@@ -136,7 +136,7 @@ def install_java_deps():
             "org.apache.commons:commons-text:1.4",
             "org.awaitility:awaitility:3.1.6",
             "org.checkerframework:checker:2.5.4",
-            "org.flywaydb:flyway-core:6.5.0",
+            "org.flywaydb:flyway-core:6.5.7",
```

Comment:
Is there anything useful in there? Your PR description presumably refers to the changes in 7.0? Not opposed to upgrading at all, quite the opposite, just curious.

Reply:
Just WIP for now -- I was trying a less intrusive minor version upgrade as second order and that looks to be fine

What we really want is the 7.x changes, but while those seem to run locally and even across a test suite they fail miserably when run on CI, presumably because of some classpath issues and maybe some race conditions?

Anyway this is still a WIP pr -- I need to do some more work on it to get things to mergeable state -- sorry for prematurely putting you as a reviewer (thought I had removed after seeing the 7 build fail spectacularly)